### DISCUSSION TITLE
How interception request to get JS upload binary file content

### DISCUSSION THREAD

**Discussion by wenbei421** [cefsharp/CefSharp#4135] [Q&A] [answered]
I'm using cefsharp to host a web page that uses js to request a custom protocol Url and request interception in C#, but I can't get the binary content of the file。
js code:
  function upload() {
            // Form Data
            var postData = new FormData();

            var fileSelect = document.getElementById("fileSelect");
            if (fileSelect.files && fileSelect.files.length == 1) {
                var file = fileSelect.files[0]
                postData.set("file", file, file.name);
            }

            axios.post('http://icreate.com/admincontroller/upload', postData)
                .then(response => {
                    console.log(response.data)
                })
                .catch(error => {
                    console.log(error)
                });
        }
C# code
public class DefaultRequestSchemeHandler : ResourceHandler
    {
      ...
     public override CefReturnValue ProcessRequestAsync(IRequest request, ICallback callback)
        {
             ...
             var postDataElements = request?.PostData?.Elements;
             ...
        }
}
post data：
![image](https://user-images.githubusercontent.com/6428597/172900924-3442825a-f03e-4dd4-a0f3-0b06f6cf16da.png)

**Comment by amaitland** [comment, marked as answer]
There's already a discussion at   #4134
